### PR TITLE
Sticky columns.

### PR DESCRIPTION
**Pull request by rpiaggio**
Kapture 2021-06-04 at 18 15 57

### CODE REVIEW

**Comment by cquiroz** [comment]
What is sticky in this context? That the first column always shows up?

File: explore/src/main/scala/explore/targeteditor/TargetSummaryTable.scala [hunk 1]
```diff
@@ -93,7 +100,10 @@ object TargetSummaryTable {
 
       val columns =
         (List(
-          column("type", _ => ()).setCell(_ => Icon("star")),
+          column("type", _ => ())
+            .setCell(_ => Icon("star"))
+            .setWidth(30),
+          // .setClassName("sticky"),
```
Comment (toddburnside) [inline explore/src/main/scala/explore/targeteditor/TargetSummaryTable.scala:106]: Leftover comment? Another one below, too.

**Review by toddburnside — APPROVED** [review]
Very cool.

**Comment by toddburnside** [comment]
> What is sticky in this context? That the first column always shows up?

**Two** columns! 😄

File: explore/src/main/scala/explore/targeteditor/TargetSummaryTable.scala (hunk 1 above)
Comment (rpiaggio) [reply explore/src/main/scala/explore/targeteditor/TargetSummaryTable.scala:106]: yup, thx. will remove

**Comment by rpiaggio** [comment]
> What is sticky in this context? That the first column always shows up?

Exactly, sticky columns cannot be scrolled out. We may want sticky headers too.

**Pull request merged**